libcontainer/cgroups/systemd/common.go

```diff
@@ -366,16 +366,24 @@ func systemdVersion(conn *systemdDbus.Conn) (int, error) {
 		}
 
 		// verStr is like "v245.4-1.fc32" (including quotes)
-		if !strings.HasPrefix(verStr, `"v`) {
-			versionErr = fmt.Errorf("can't parse version %s", verStr)
+		if strings.HasPrefix(verStr, `"v`) {
```

Comment:
instead of using two separate branches, would it perhaps be an option to unconditionally remove leading/trailing quotes (`"`) and `v`;

```
strings.Trim(verStr, `v"`)
```

and then look for the version before the (optional) dot?